frontend/src/page/About.tsx

```diff
@@ -1,8 +1,16 @@
 import React from "react";
 import { useTranslation, Trans } from "react-i18next";
+import { Root } from "../layout/Root";
+import type { Route } from "../router";
 
 
-export const About: React.FC = () => {
+export const AboutRoute: Route<void> = {
+    path: "/about",
+    prepare: () => {},
+    render: () => <Root><About /></Root>,
```

Comment:
Because routes decide on what navigation to show and the navigation has to be passed to route. Doesn't happen in this PR, but in a later one. That is one of the main reasons for this change.

Reply:
Aah ok, I guess I can suspend my doubt a little more. :wink: